automated install/basic-install.sh

```diff
@@ -948,14 +958,31 @@ setAdminFlag() {
       "On (Recommended)")
         echo -e "  ${INFO} Web Interface On"
         # Set it to true
-        INSTALL_WEB=true
+        INSTALL_WEB_INTERFACE=true
         ;;
       Off)
         echo -e "  ${INFO} Web Interface Off"
         # or false
-        INSTALL_WEB=false
+        INSTALL_WEB_INTERFACE=false
         ;;
     esac
+
+  # Request user to install web server, if --disable-install-webserver has not been used (INSTALL_WEB_SERVER=true is default).
+  if [[ "${INSTALL_WEB_SERVER}" == true ]]; then
+    WebToggleCommand=(whiptail --separate-output --radiolist "Do you wish to install the web server (lighttpd)?\n\nNB: If you disable this, and, do not have an existing webserver installed, the web interface will not function." ${r} ${c} 6)
```

Comment:
@stickler-ci 

> ${r} ${c} Quote to prevent word splitting, or split robustly with mapfile or read -a.

Values are integer, no word splitting will occur.

> Backslash is literal in "\n". Prefer explicit escaping: "\n".

Disagree: https://github.com/koalaman/shellcheck/wiki/SC1117#exceptions
Been using single backslash in DietPi for years.

However, to keep the bot happy 🍼 , i'll send commit with required changes. 😃

Reply:
Yeah, there are probably some shellcheck rules we can disable, but for consistency it's nice to have something that enforces codestyle :) Will check out your new whiptail in a bit